packages/expo-image-picker/CHANGELOG.md

```diff
@@ -7,3 +7,5 @@
 ### 🎉 New features
 
 ### 🐛 Bug fixes
+
+- Fixed exception when calling `getCameraPermissionsAsync` on Web. ([#7498](https://github.com/expo/expo/pull/7498) by [@IjzerenHein](https://github.com/IjzerenHein))
```

Comment:
```suggestion
- Fixed exception when calling `ImagePicker.getCameraPermissionsAsync` on Web. ([#7498](https://github.com/expo/expo/pull/7498) by [@IjzerenHein](https://github.com/IjzerenHein))
```

Reply:
Summon @tsapeta 🤔 Do you think we should always use `Module.function` notation in these `CHANGELOG`s?